example/ck_tile/03_gemm/universal_gemm.cpp

```diff
@@ -60,10 +60,13 @@ float gemm_calc(const ck_tile::GemmHostArgs& args, const ck_tile::stream_config&
 
     float ave_time{0};
 
-    const auto Run = [&](const auto has_hot_loop_, const auto tail_number_) {
-        constexpr bool has_hot_loop_v = has_hot_loop_.value;
-        constexpr auto tail_number_v  = tail_number_.value;
-        constexpr auto scheduler      = GEMM_PIPELINE_SCHEDULER;
+    const auto Run_with_k_batch = [&](const auto has_hot_loop_,
```

Comment:
I'd actually swap names. And here use `Run` and in the other place eg. `RunSplitk`